docs/examples/move/nft_marketplace/README.md

```diff
@@ -0,0 +1,119 @@
+# Marketplace Guide
+
+The `nft_marketplace.move` module provides a straightforward implementation of a marketplace extension. To utilize it, follow the steps outlined below.
+The `item_for_market.move` contains mocked item data for use within the marketplace.
+The `rental_extension.move` is an extension adds functionality to enable item rentals.
+
+
+## Steps to Use the Marketplace
+
+### 1. Connect to the Network
+Connect to the IOTA network (e.g., using a faucet to obtain tokens).
+
+### 2. Install Kiosk
+Run the following command to install the Kiosk module:
```

Comment:
What does it mean to "install"? Pls, describe that this function "default" creates a kiosk objects and shares it and also creates a owner cap and transfers it to the caller.

Reply:
Done